pkg/server/cloud/aws/instances.go

```diff
@@ -231,6 +231,17 @@ func bootImageSpecToDescribeImagesInput(spec cloud.BootImageSpec) *ec2.DescribeI
 	return input
 }
 
+func getRootDeviceVolumeSize(blockDevices []*ec2.BlockDeviceMapping, rootDeviceName string) int32 {
+	var rootDiskSize int32
+	for _, blockDevice := range blockDevices {
+		if *blockDevice.DeviceName == rootDeviceName && blockDevice.Ebs != nil {
```

Comment:
I know that I suggested this change but it also makes the mistake of dereferencing (the possibly nil) `blockDevice.DeviceName`.  To keep everything consistent, lets use `aws.StringValue` here as well.

Reply:
:+1: fixed